### ISSUE TITLE
Incorrect highlighting when using Pygment with linenums

### ISSUE THREAD

**Issue by bromine0x23** [asciidoctor/asciidoctor#3145]
Hello, I'm using AsciiDoctor with Pygment highlighting.
When I using `linenums` with a dark theme (e.g. `monokai`) to a plain text code block, I got an incorrect result: the plain text not using dark theme style but the default style.

A sample case is like that:
```asciidoctor
:source-highlighter: pygments
:pygments-style: monokai

[source,pseudocode]
----
if right < left then
    p := PARTITION(A, left, right)
    QUICKSORT(A, left, p - 1)
    QUICKSORT(A, p + 1, right)
----

[source,pseudocode,linenums]
----
if right < left then
    p := PARTITION(A, left, right)
    QUICKSORT(A, left, p - 1)
    QUICKSORT(A, p + 1, right)
----
```

![asciidoctor-pygments-monokai](https://user-images.githubusercontent.com/5657337/54254210-ebc87900-458d-11e9-8f0a-94700d413da4.png)

AsciiDoctor: 1.5.8
Ruby: 2.5.3p105
pygments.rb: 1.2.1
Python: 2.7.15

**Comment by mojavelinux** [comment]
This was a known issue in 1.5.8 that has been addressed in the upcoming 2.0.0 release. Please try the 2.0.0.rc.2 gem and you'll see that it has been resolved.

It was addressed by #3016.

**Comment by bromine0x23** [comment]
Thanks for the answers. 
But is there any Docker image for that version, since I'm generating documents by CI tools on Docker.

**Comment by mojavelinux** [comment]
No, the Docker image will be available once the final version (2.0.0) is released.

**Comment by bromine0x23** [comment]
Got it.
